### PR TITLE
Add com.github.unrud.VideoDownloader

### PR DESCRIPTION
**Pull request by Unrud**

### CODE REVIEW

**Comment by Unrud** [comment]
bot, build com.github.unrud.VideoDownloader

**Comment by Unrud** [comment]
> eu-strip: while opening '.../rofiles-LnbK55/files/lib/libtcl8.6.so': Permission denied

Is this a bug in Flatpak?

**Comment by Unrud** [comment]
bot, build com.github.unrud.VideoDownloader

File: com.github.unrud.VideoDownloader.yaml [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+      - /etc
+      - /share/doc
+      - /share/man
+    sources:
+      - type: archive
+        url: https://youtube-dl.org/downloads/2019.05.20/youtube-dl-2019.05.20.tar.gz
+        sha256: a8a6cdf24bcbfa3d3ac9ff9f7efad917c707f9834876b4269036e379b76bbca3
+
+  - name: video-downloader
+    builddir: true
```
Comment (bilelmoussaoui) [inline com.github.unrud.VideoDownloader.yaml]:
```suggestion

```

**Comment by Unrud** [comment]
bot, build com.github.unrud.VideoDownloader

**Comment by Unrud** [comment]
bot, build com.github.unrud.VideoDownloader

**Comment by Unrud** [comment]
bot, build com.github.unrud.VideoDownloader

**Comment by nedrichards** [comment]
bot, build com.github.unrud.VideoDownloader

**Comment by Unrud** [comment]
bot, build com.github.unrud.VideoDownloader

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.github.unrud.VideoDownloader

**Pull request closed without merging**